Makefile

```diff
@@ -46,4 +48,12 @@ release:
 	opam publish submit $(NAME_VERSION)
 	rm -rf $(NAME_VERSION)
 
+install:
+	ocamlfind remove ppx_deriving
+	grep -E '^[[:space:]]+' ppx_deriving.install | \
```

Comment:
I've used this Makefile against MSYS (not cygwin), and as ocamlbuild needs a bash in the path to work well (it was a harsh discovery), I don't think we can completely get rid of all Unix replacement platform on windows :-/